src/Thujohn/Twitter/TwitterServiceProvider.php

```diff
@@ -62,4 +62,4 @@ public function provides()
 		return ['ttwitter'];
```

Comment:
Does the PHP version need to be updated now?
I opened #146 

Would a PHP version update justify a minor version change as opposed to a patch?

Thanks,
Greg